src/GraphQl/Serializer/ItemNormalizer.php

```diff
@@ -110,6 +111,25 @@ public function normalize(mixed $object, string $format = null, array $context =
      */
     protected function normalizeCollectionOfRelations(ApiProperty $propertyMetadata, iterable $attributeValue, string $resourceClass, ?string $format, array $context): array
     {
+        if ('item_query' === $context['graphql_operation_name'] && $this->resourceMetadataCollectionFactory) {
+            // check for nested collection
+            $operation = $this->resourceMetadataCollectionFactory->create($resourceClass)->getOperation('collection_query');
```

Comment:
```suggestion
            $operation = $this->resourceMetadataCollectionFactory->create($resourceClass)->getOperation(forceCollection: true);
``` 
mhh maybe we need something to fetch only graphql operations?

Reply:
using `forceCollection: true` is not identical. It returns the `GetCollection` and not the `QueryCollection`

Reply:
Then we need to loop over the resources, I'm willing to add an argument `forceGraphQl: true` to the `getOperation` as I feel this is missing right now.

Reply:
Added argument `forceGraphQl`. Works fine